pkgs/development/python-modules/pyside2/default.nix

```
@@ -79,5 +79,6 @@ stdenv.mkDerivation rec {
     license = licenses.lgpl21;
     homepage = "https://wiki.qt.io/Qt_for_Python";
     maintainers = with maintainers; [ gebner ];
+    broken = stdenv.isDarwin;
```

```suggestion
    platforms = platforms.all;
    broken = stdenv.isDarwin;
```

It surprises me that the failure was not tracked by Hydra: https://hydra.nixos.org/eval/1807459?filter=pyside2&compare=1807454&full=

---

Good spot. Done